### ISSUE TITLE
Connection not kept alive using SSH in Remmina 1.2

### ISSUE THREAD

**Issue by e-alfred** [FreeRDP/Remmina#985]
Hello,

I am running around 20 SSH sessions in Remmina 1.2 but most of them get closed after some time. In older versions, this problem did not exist, connections were kept alive for as long as the client run. I tried to set the "ServerAliveInterval" to 60 but it won't get parsed by Remmina claiming it is an unsupported option. How can I keep connections alive in Remmina 1.2?

My system specs:

```
Client (OS name and version): Kubuntu 14.04.5
Remmina version (remmina --version): 1.2.0-rcgit-15
Desktop environment (GNOME, Unity, KDE, ..): KDE 4.14
Connecting to (OS and version): Ubuntu, Debian, Centos...
Connecting via (RDP, VNC, ...): SSH
```

Thank you
##

**Comment by antenore** [comment]
Can you check if you have any TMOUT or TIMEOUT environment variable set on your remote host?

In the meanwhile I've linked this issue with one opened by @mreichardt95 on libssh https://red.libssh.org/issues/239  

This is duplicate of #975

**Comment by e-alfred** [comment]
I have no timeout values set. It worked well with an older version of Remmina before I upgraded with the PPA to 1.2. There were no changes to the different SSH servers.

**Comment by Chugajstyr** [comment]
I`ve same problem - after 5 min SSH session is "frozen" alike (but cursor in console is blinking) despite on parameter "ServerAliveInterval" in `~/.ssh/config`. My system specs:

`Client (OS name and version): Linux Mint 17.3 "Rosa" (Ubuntu 14.04 based)
Remmina version (remmina --version): 1.2.0-rcgit-18
Desktop environment (GNOME, Unity, KDE, ..): MATE 1.12
Connecting to (OS and version): Centos
Connecting via (RDP, VNC, ...): SSH`

**Comment by antenore** [comment]
Keepalive in Remmina is implemented, to understand what happens I need some more info please.

1) Which version of libssh do you have?
2) What happens if you set the TMOUT and TIMEOUT to 0 on your server?
3) Can you open the Remmina DEBUG window and look for any [SSH] TCP messages?

The TMOUT and TIMEOUT variables are often set for security reasons (compliance rules mainly), so it's something that may have been changed recently (months/few years).

**Comment by antenore** [comment]
Resolved via #1553
